cmd/cstor-volume-mgmt/controller/common/common_test.go

```diff
@@ -0,0 +1,33 @@
+package common
+
+import (
+	"testing"
+	"time"
+
+	openebsFakeClientset "github.com/openebs/maya/pkg/client/clientset/versioned/fake"
+)
+
+// TestCheckForCStorVolumeCR validates if CStorVolume CR operations
+// can be done.
+func TestCheckForCStorVolumeCR(t *testing.T) {
+	fakeOpenebsClient := openebsFakeClientset.NewSimpleClientset()
+	done := make(chan bool)
+
```

Comment:
goroutine leakage `defer close(done)`

Reply:
Sure. Will close the channel. 
FYI: leakage will not happen as go is garbage collected. But still it is a good practice to close an open channel. For files, close is absolutely necessary whereas for channels it is not so. We need to close a channel if the code checks for a closure of a channel and does some operation based on it.
But anyway, I will close the channel as it will be neat and avoids any further confusion.